app/routines/update_user_salesforce_info.rb

```diff
@@ -225,7 +225,7 @@ def cache_contact_data_in_user!(contact, user)
         :other_school_type
       end
 
-      if contact.adoption_status != nil
+      unless contact.adoption_status.nil? || contact.adoption_status.empty?
```

Comment:
You can call `.blank?` instead of `.nil?` and `.empty?`

Reply:
I haven't done a complete review, just saw this and thought I should mention it sooner rather than later.

Reply:
I didn't think I could use `.blank?` because contact is not an ActiveRecord object. It is an object from the openstax-salesforce Gem.

Reply:
But you're checking `adoption_status` not `contact`, so it doesn't matter what kind of object `contact` is :).

Reply:
So `.blank` is available on any field on any object? I didn't know that. I read it was only for ActiveRecords. I can make that change.

Reply:
👍 Rails adds it to every object. 
https://apidock.com/rails/Object/blank%3f